build/npm/clarity-icons.json

```diff
@@ -3,7 +3,8 @@
     "version": "/* @echo VERSION */",
     "description": "Custom Element Icons for Clarity",
     "homepage": "https://vmware.github.io/clarity/",
-    "main": "clarity-icons.min.js",
+    "main": "index.js",
+    "types": "index.d.ts",
```

Comment:
It's basically pointing to the proper declaration file, which is index.d.ts in this case. We don't have to specify this `types` field value as the declaration is already named `index.d.ts`. However, typescript documentation says it's it is advisable to do so.